python/ray/autoscaler/aws/config.py

```diff
@@ -319,6 +294,39 @@ def _check_ami(config):
                         region=region))
 
 
+def _upsert_security_groups(config):
+    group_name = SECURITY_GROUP_TEMPLATE.format(config["cluster_name"])
+
+    security_groups = _get_or_create_vpc_security_groups(config, group_name)
+    _upsert_security_group_rules(config, security_groups)
+
+    return security_groups
+
+
+def _get_or_create_vpc_security_groups(conf, group_name):
```

Comment:
I'm not too opinionated about this, but would probably prefer "sg_name" over "group_name" or "cluster_name" to be more specific about exactly what this name refers to (reusing the convention of "sg" for "security group"). Also, even though the security group name used here contains the cluster name, I wouldn't want to give the impression that the two are equal (e.g. the current default security group name is "ray-autoscaler-{cluster_name}").

However, taking a second look at this, we don't even need to pass this as an input argument since this is the only method that uses it. We can instead generate the security group name directly within _get_or_create_vpc_security_groups() as:

``` python
    sg_name = SECURITY_GROUP_TEMPLATE.format(conf["cluster_name"])
```